### PR TITLE
chore: add template issues

### PR DESCRIPTION
**Pull request by pedronauck**
This pull request adds two types of issues templates, one for bugs and another for features.

### CODE REVIEW

**Review by luizstacio — CHANGES_REQUESTED** [review]
Let's split the refactoring of the CI and the new template on another PR

**Comment by pedronauck** [comment]
splitted ✅

File: .github/ISSUE_TEMPLATE/feature.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: 🚀 Feature request
+description: Suggest an idea for this project.
+labels: ["feature"]
```
Comment (adlerjohn) [inline .github/ISSUE_TEMPLATE/feature.yaml:3]: This label doesn't actually exist. Should change to

```suggestion
labels: ["enhancement"]
```

Comment (pedronauck) [reply .github/ISSUE_TEMPLATE/feature.yaml]: ✅ @adlerjohn

**Review by luizstacio — APPROVED** [review]

**Pull request merged**